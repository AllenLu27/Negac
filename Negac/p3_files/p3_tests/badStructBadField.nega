int foo;
int foo () {
    if(invalidDecl) {
        return 0;
    }
}
struct Foo {
    int a;
};
struct Foo {
    void a;
};
