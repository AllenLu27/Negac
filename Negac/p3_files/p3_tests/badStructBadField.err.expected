FATAL [2,5]: Multiply declared identifier
FATAL [3,8]: Undeclared identifier
FATAL [10,8]: Multiply declared identifier
FATAL [11,10]: Invalid type in declaration
Semantic analysis failed
